suite.py

```diff
@@ -51,6 +55,14 @@ def __call__(self, event, context):
                 conclusion=NEUTRAL
             )
 
+    @staticmethod
+    def find_config_file(path, regexpr):
+        """Return first file matching the regular expression."""
+        for base_dir, dir_names, filenames in os.walk(path):
+            for filename in filenames:
+                if regexpr.match(filename):
+                    return os.path.join(path, filename)
```

Comment:
That seems a bit inefficient.. would be better to just check if a file exists, instead looking at all files.

I would also not make it ignore case.

Reply:
That is a good point, but would currently mean about 24 different names and it must be case insensitive which makes it around 64**24 possible file names. This would be more files than some file systems allow. Therefore I think using `os.walk` is somewhat efficient and also how `glob` works if I am not mistaken.